akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowSpec.scala

```diff
@@ -593,8 +593,21 @@ class FlowSpec extends StreamSpec(ConfigFactory.parseString("akka.actor.debug.re
       import Attributes._
       val f: Flow[Int, Int, NotUsed] = Flow[Int].map(_ + 1).async.addAttributes(none).named("name")
 
-      f.module.attributes.getFirst[Name] shouldEqual Some(Name("name"))
-      f.module.attributes.getFirst[Attributes.AsyncBoundary.type] shouldEqual Some(AsyncBoundary)
+      f.module.attributes.get[Name] shouldEqual Some(Name("name"))
+      f.module.attributes.get[Attributes.AsyncBoundary.type] shouldEqual Some(AsyncBoundary)
```

Comment:
Was these changes a consequence of the fix? (least specific vs most specific attribute)